network/src/protocols/rpc/mod.rs

```diff
@@ -318,6 +319,8 @@ impl InboundRpcs {
 }
 
 /// `OutboundRpcs` handles new outbound rpc requests made from the application layer.
+///
+/// There is one `OutboundRpcs` handler per connection.
```

Comment:
Perhaps per Peer is more appropriate?  Or maybe just drop this comment since we have only one peer per connection?  And I'm not sure if we had a multiple connections to a single peer in the future, we'd need multiple outbound RPC handlers.